suite-native/formatters/src/components/FiatBalanceFormatter.tsx

```diff
@@ -53,6 +52,26 @@ export const FiatBalanceFormatter = ({
                 isDiscreetText
                 isForcedDiscreetMode={isForcedDiscreetMode}
             />
+        </>
+    );
+
+    const currencyElement = <Text variant="titleSmall">{currencySymbol}</Text>;
+
+    const isBehind = currencySymbol === 'sat';
+
+    return (
+        <Box flexDirection="row" alignItems="flex-end" flexShrink={1} testID={testID}>
```

Comment:
Why not use flexbox? 
```suggestion
        <Box flexDirection={isSuffix ? "row" : "row-reverse"} alignItems="flex-end" flexShrink={1} testID={testID}>
```

Reply:
It is now a bit different so probably not relevant anymore.